core/src/main/scala/org/apache/spark/storage/TachyonBlockManager.scala

```diff
@@ -120,6 +134,16 @@ private[spark] class TachyonBlockManager() extends ExternalBlockManager with Log
     }
   }
 
+  override def getValues(blockId: BlockId): Option[Iterator[_]] = {
+    val file = getFile(blockId)
+    if (file == null || file.getLocationHosts().size() == 0) {
+      return None
+    }
+    val is = file.getInStream(ReadType.CACHE)
+    assert (is != null)
```

Comment:
I think the code is pretty inconsistent about this, but do you mean `assert` here? if assertions aren't on is it OK if this isn't checked? probably, just asking.  Also tiny nit: you don't need `return` below.